balancer-js/src/modules/contracts/contracts.module.ts

```diff
@@ -46,6 +47,7 @@ export interface ContractInstances {
   multicall: Contract;
   relayerV3?: Contract;
   relayerV4?: Contract;
+  relayerV5?: RelayerV5;
```

Comment:
I see that for some cases we're declaring as a generic Contract and others we're using the specific contract type (e.g. Vault, RelayerV5, ...).
My general feeling is that we should lean toward more specific types, but I'm not fully aware of the implications here.
Does anyone have strong opinions on this?

Reply:
Nice that we'll be able to stick with one Relayer! 😃 
But I'd still like to get a feeling for declaring generic vs specific Contract type. I mean, regarding all pool factories as well. Should we keep them as is or update to what typechain generates?

Reply:
so relayerV3, relayerV4 can be removed?

Reply:
Should be able to remove V3/V4 after front-end remove Zap references. Then we stick to a single (most recent) Relayer.